.github/workflows/hermetic_library_generation.yaml

```diff
@@ -17,10 +17,14 @@ name: Hermetic library generation upon generation config change through pull req
 on:
   pull_request:
 
+env:
+  HEAD_REF: ${{ github.head_ref }}
+  REPO_FULL_NAME: ${{ github.event.pull_request.head.repo.full_name }}
+
 jobs:
   library_generation:
     # skip pull requests coming from a forked repository
-    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Could we separate these unrelated commits?

Reply:
I never made these commits. Seem to be rebase/merge artifacts?